Sources/FirebaseAndroid/CMakeLists.txt

```diff
@@ -7,3 +7,7 @@ target_include_directories(FirebaseAndroidJNI PUBLIC
 target_link_libraries(FirebaseAndroidJNI PRIVATE
   log)
 
+add_jar(SwiftFirebase
+          Native.java
+        INCLUDE_JARS
+          $ENV{ANDROID_SDK_ROOT}/platforms/android-${ANDROID_NATIVE_API_LEVEL}/android.jar)
```

Comment:
Is the indentation of the last line as intended? It looks like a really long line that wraps around the `$ENV...`?

Reply:
Yes, it is a single entry (it is a path).